index.ts

```diff
@@ -65,6 +67,11 @@ async function main() {
                     // - pricing
                     // - repository name (devpool issue body contains a partner project issue URL)
                     // - bounty hunter assigned/unassigned an issue
+                    const issuePriceLabel  = getIssuePriceLabel(projectIssue) 
```

Comment:
If an issue exists in a partner's project but it doesn't exist in the devpool then [here](https://github.com/ubiquity/devpool-directory/blob/82603610e4e51ec604ad03732021f40868450a16/index.ts#L103) it will be created with the `Pricing: not set` label. As far as I understand the expected behaviour is that **all** issues without a price should be skipped from publishing to the devpool.

Reply:
@rndquu Sorry misread the code, updated now. Please check.